notification/store.go

```diff
@@ -266,41 +268,41 @@ func (db *DB) SendContactMethodVerification(ctx context.Context, id string, rese
 	return tx.Commit()
 }
 
-func (db *DB) VerifyContactMethod(ctx context.Context, cmID string, code int) ([]string, error) {
+func (db *DB) VerifyContactMethod(ctx context.Context, cmID string, _code string) error {
 	userID, err := db.cmUserID(ctx, cmID)
 	if err != nil {
-		return nil, err
+		return err
+	}
+
+	code, err := strconv.ParseInt(_code, 10, 64)
```

Comment:
We can do more granular error checking after we change these to int, but the Store's job is to just wrap the DB, so let's move the `Range` check to the GraphQL layer.